package.json

```diff
@@ -48,7 +48,7 @@
   "main": "webpack.config.js",
   "keywords": [],
   "dependencies": {
-    "identity-style-guide": "^0.2.4",
+    "identity-style-guide": "18f/identity-style-guide#bh-add-app-styles",
```

Comment:
Yep that was my plan - I'll merge the `bh-add-app-styles` branch into master on `identity-style-guide`, publish to npm, then update this PR and the one for `identity-idp` to just point to the normal npm package before I merge them. Not sure if there are better workflows moving forward though, you have any thoughts?